src/openloco/ui/WindowManager.cpp

```diff
@@ -273,7 +269,63 @@ namespace openloco::ui::WindowManager
     // 0x004C6118
     void update()
     {
-        call(0x004C6118);
+        _tooltipNotShownTicks = _tooltipNotShownTicks + time_since_last_tick;
+
+        // Has to do with graphics mode switching. Remove?
+        _9DA3D4 = _9DA3D4 + 1;
```

Comment:
9DA3D4 does not seem to be referred to outside of this sub, save for an initialisation. If the logic is indeed obsolete, you can get rid of it entirely, afaict.